pages/course/[courseId]/index.tsx

```diff
@@ -27,22 +35,24 @@ export default function Details() {
   const course = coursesById[0];
 
   //TODO: change later, when implementing the services into this side
-  const chapters = [
-    "Chapter 1",
-    "Chapter 2",
-    "Chapter 3",
-    "Chapter 4",
-    "Chapter 5",
-    "Chapter 6",
-    "Chapter 7",
-    "Chapter 8",
-    "Chapter 9",
-    "Chapter 10",
-  ];
+  const chapters = course.chapters;
   const knowledge = 88;
   const understanding = 50;
   const analyses = 40;
   const usage = 22;
+  let viewablechapters = [];
+
+  for (let index = 0; index < chapters.length; index++) {
```

Comment:
You can also use `for (let chapter in chapters) {...}` (if you didn't know).
Or even `const viewablechapters = chapters.filter(chapter => ...)`.

Just a comment though. This is also fine for me.

Reply:
The first option: I forgot about that possibility, thanks for the reminder.
The second option: I tried that at first, but I did not get it to work, so I used the old school method.